SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -60,46 +60,35 @@ public async Task<IActionResult> AddUser([FromBody] User user)
                 return new BadRequestObjectResult("user cannot be null");
             }
         }
-        // PUT: api/User/
+        // PUT: api/User
         [HttpPut("{id}")]
         public async Task<IActionResult> UpdateUser(int id, [FromBody] User user)
```

Comment:
All of my above changes can be applied to this function as well

Reply:
I made afew changes to match the previous ones